tests/Backend/CommonPart1/CreateTableTest.php

```diff
@@ -140,7 +140,11 @@ public function testTableFromEmptyFileShouldNotBeCreated()
 
     public function testTableCreateFromNotUploadedFileShouldThrowError()
     {
-        $file = $this->_client->prepareFileUpload((new FileUploadOptions())->setFileName('missing'));
+        $file = $this->_client->prepareFileUpload(
+            (new FileUploadOptions())
+                ->setFileName('missing')
+                ->setFederationToken(true)
```

Comment:
Koukal jsem co to vlastně je ten federation token a není to v KBC nikde moc vysvětlené. Zmínka je jen v https://developers.keboola.com/integrate/storage/api/import-export/#upload-files-using-http

Reply:
Federation token znamená še to vrátí dočasné S3 credentials. Nově bude povinný a až se zbavíme těch pár legacy form uploadů tak se to tak bude chovat vždy viz. postup https://github.com/keboola/connection/issues/1448

Popsaný je tady https://developers.keboola.com/integrate/storage/api/import-export/#manually-uploading-a-file a tady https://keboola.docs.apiary.io/#reference/files/upload-file/create-file-resource